package.json

```diff
@@ -386,6 +387,26 @@
           "finance.allocations.item.post"
         ]
       },
+      {
+        "permissionName": "ui-finance.fund-update-logs.view",
+        "displayName": "Finance: View batch allocation logs",
+        "description": "",
+        "visible": true,
+        "subPermissions": [
+          "ui-finance.fund-budget.view",
+          "finance.fund-update-logs.collection.get"
```

Comment:
I suppose `finance.fund-update-logs.item.get` should be also listed here since a user needs to get log details